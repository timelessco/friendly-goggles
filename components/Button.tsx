import React, { forwardRef, PropsWithChildren } from "react";

export type ButtonProps = PropsWithChildren<{}>;

export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
  (props, ref) => {
    return <button ref={ref} {...props} />;
  },
);
