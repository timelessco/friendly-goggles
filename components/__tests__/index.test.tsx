import { render } from "@testing-library/react";

import { Button } from "../Button";

it("renders CopyButton unchanged", () => {
  const { container } = render(<Button>Button</Button>);
  expect(container).toMatchSnapshot();
});
