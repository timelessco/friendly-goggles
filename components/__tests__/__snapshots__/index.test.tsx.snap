// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CopyButton unchanged 1`] = `
<div>
  <button>
    Button
  </button>
</div>
`;
